김은지/짝지어-제거하기.js

```diff
@@ -0,0 +1,17 @@
+function checkRemovable(s) {
+  const stack = [];
+
+  for (const char of s) {
+    if (stack.at(-1) !== char) {
```

Comment:
```suggestion
    if (stack[stack.length - 1] !== char) {
```
<img width="200" alt="image" src="https://github.com/user-attachments/assets/973659fa-c7bb-49ec-a6b8-1a92781323bf">

이렇게 바꾸니 효율성 테스트 1 케이스 시간 확 줄어든 게 보이네용!

Reply:
이런 차이가 있는거 이번 기회에 알아갑니다..!!